tests/common/devices/multi_asic.py

```diff
@@ -43,6 +43,9 @@ def __init__(self, ansible_adhoc, hostname):
 
         self.critical_services_tracking_list()
 
+    def __repr__(self):
+            return self.hostname
```

Comment:
Is it better to return something like `'<MultiAsicSonicHost {}>'.format(self.hostname)`?

Reply:
Good idea, will change